Vic2ToHoI4/Source/HOI4World/States/HoI4State.cpp

```diff
@@ -138,6 +138,21 @@ void HoI4::State::convertControlledProvinces(const std::vector<std::pair<int, st
 }
 
 
+void HoI4::State::setControlledProvince(int provinceNum, const std::string& country)
```

Comment:
```suggestion
void HoI4::State::setControlledProvince(const int provinceNum, const std::string& country)
```

Reply:
Even though the checkers suggest it, I'm really not fond of const int in function signatures.